### ISSUE TITLE
[opencv] No hidden protobuf

### ISSUE THREAD

**Issue by alexlee728** [microsoft/vcpkg#39224]
No hidden protobuf

```
vcpkg/installed/x64-linux/lib$ readelf -Ws libopencv_dnn4.a | grep protobuf
     5: 0000000000000000     0 SECTION LOCAL  DEFAULT  474 .text._ZN6google8protobuf11MessageLite25OnDemandRegisterArenaDtorEPNS0_5ArenaE
   293: 0000000000000000     0 SECTION LOCAL  DEFAULT  834 .text._ZNK12opencv_caffe9BlobShape18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   389: 0000000000000000     0 SECTION LOCAL  DEFAULT 1022 .text._ZNK12opencv_caffe15BlobProtoVector18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   390: 0000000000000000     0 SECTION LOCAL  DEFAULT 1024 .text._ZNK12opencv_caffe14InputParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   432: 0000000000000000     0 SECTION LOCAL  DEFAULT 1108 .text._ZN6google8protobuf8internal7ReadTagEPKcPjj.isra.33
   433: 0000000000000000    63 FUNC    LOCAL  DEFAULT 1108 _ZN6google8protobuf8internal7ReadTagEPKcPjj.isra.33
   434: 0000000000000000     0 SECTION LOCAL  DEFAULT 1110 .text._ZN6google8protobuf13RepeatedFieldIfE3AddERKf.isra.447
   435: 0000000000000000    94 FUNC    LOCAL  DEFAULT 1110 _ZN6google8protobuf13RepeatedFieldIfE3AddERKf.isra.447
   436: 0000000000000000     0 SECTION LOCAL  DEFAULT 1112 .text._ZN6google8protobuf13RepeatedFieldIjE3AddERKj.isra.449
   437: 0000000000000000    90 FUNC    LOCAL  DEFAULT 1112 _ZN6google8protobuf13RepeatedFieldIjE3AddERKj.isra.449
   439: 0000000000000000     0 SECTION LOCAL  DEFAULT 1115 .text._ZN6google8protobuf13RepeatedFieldIfEC2ERKS2_.part.506
   440: 0000000000000000    50 FUNC    LOCAL  DEFAULT 1115 _ZN6google8protobuf13RepeatedFieldIfEC2ERKS2_.part.506
   441: 0000000000000000     0 SECTION LOCAL  DEFAULT 1117 .text._ZNK12opencv_caffe18ParameterParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   442: 0000000000000000     0 SECTION LOCAL  DEFAULT 1119 .text._ZNK12opencv_caffe18ThresholdParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   443: 0000000000000000     0 SECTION LOCAL  DEFAULT 1121 .text._ZNK12opencv_caffe12ELUParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   444: 0000000000000000     0 SECTION LOCAL  DEFAULT 1123 .text._ZNK12opencv_caffe12ExpParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   445: 0000000000000000     0 SECTION LOCAL  DEFAULT 1125 .text._ZNK12opencv_caffe12LogParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   446: 0000000000000000     0 SECTION LOCAL  DEFAULT 1127 .text._ZNK12opencv_caffe14PowerParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   448: 0000000000000000     0 SECTION LOCAL  DEFAULT 1130 .text._ZN6google8protobuf8internal21arena_destruct_objectINS1_16InternalMetadata9ContainerINS0_15UnknownFieldSetEEEEEvPv
   449: 0000000000000000     0 SECTION LOCAL  DEFAULT 1132 .gcc_except_table._ZN6google8protobuf8internal21arena_destruct_objectINS1_16InternalMetadata9ContainerINS0_15UnknownFieldSetEEEEEvPv
   452: 0000000000000000     0 SECTION LOCAL  DEFAULT 1136 .text._ZNK12opencv_caffe16PermuteParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   455: 0000000000000000     0 SECTION LOCAL  DEFAULT 1141 .text._ZNK12opencv_caffe18DummyDataParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   459: 0000000000000000     0 SECTION LOCAL  DEFAULT 1149 .text._ZNK12opencv_caffe13TanHParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   460: 0000000000000000     0 SECTION LOCAL  DEFAULT 1151 .text._ZNK12opencv_caffe16SigmoidParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   461: 0000000000000000     0 SECTION LOCAL  DEFAULT 1153 .text._ZNK12opencv_caffe18HingeLossParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   465: 0000000000000000     0 SECTION LOCAL  DEFAULT 1161 .text._ZN6google8protobuf2io19EpsCopyOutputStream23WriteStringMaybeAliasedEjRKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPh
   466: 0000000000000000     0 SECTION LOCAL  DEFAULT 1163 .text._ZNK12opencv_caffe19HDF5OutputParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   467: 0000000000000000     0 SECTION LOCAL  DEFAULT 1165 .text._ZNK12opencv_caffe21InfogainLossParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   468: 0000000000000000     0 SECTION LOCAL  DEFAULT 1167 .text._ZN6google8protobuf2io17CodedOutputStream20WriteVarint32ToArrayEjPh
   469: 0000000000000000     0 SECTION LOCAL  DEFAULT 1168 .text._ZNK12opencv_caffe22NormalizeBBoxParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   470: 0000000000000000     0 SECTION LOCAL  DEFAULT 1170 .text._ZNK12opencv_caffe17PriorBoxParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   471: 0000000000000000     0 SECTION LOCAL  DEFAULT 1172 .text._ZNK12opencv_caffe23TransformationParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   472: 0000000000000000     0 SECTION LOCAL  DEFAULT 1174 .text._ZNK12opencv_caffe18BatchNormParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   473: 0000000000000000     0 SECTION LOCAL  DEFAULT 1176 .text._ZNK12opencv_caffe24ContrastiveLossParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   474: 0000000000000000     0 SECTION LOCAL  DEFAULT 1178 .text._ZNK12opencv_caffe19SaveOutputParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   475: 0000000000000000     0 SECTION LOCAL  DEFAULT 1180 .text._ZNK12opencv_caffe16DropoutParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   476: 0000000000000000     0 SECTION LOCAL  DEFAULT 1182 .text._ZNK12opencv_caffe14EmbedParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   477: 0000000000000000     0 SECTION LOCAL  DEFAULT 1184 .text._ZNK12opencv_caffe17HDF5DataParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   478: 0000000000000000     0 SECTION LOCAL  DEFAULT 1186 .text._ZNK12opencv_caffe18ImageDataParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   479: 0000000000000000     0 SECTION LOCAL  DEFAULT 1188 .text._ZNK12opencv_caffe19MemoryDataParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   480: 0000000000000000     0 SECTION LOCAL  DEFAULT 1190 .text._ZNK12opencv_caffe12MVNParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   481: 0000000000000000     0 SECTION LOCAL  DEFAULT 1192 .text._ZNK12opencv_caffe15PythonParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   482: 0000000000000000     0 SECTION LOCAL  DEFAULT 1194 .text._ZNK12opencv_caffe18RecurrentParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   483: 0000000000000000     0 SECTION LOCAL  DEFAULT 1196 .text._ZNK12opencv_caffe19WindowDataParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   484: 0000000000000000     0 SECTION LOCAL  DEFAULT 1198 .text._ZNK12opencv_caffe14PReLUParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   485: 0000000000000000     0 SECTION LOCAL  DEFAULT 1200 .text._ZNK12opencv_caffe19ROIPoolingParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   486: 0000000000000000     0 SECTION LOCAL  DEFAULT 1202 .text._ZNK12opencv_caffe17ProposalParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   487: 0000000000000000     0 SECTION LOCAL  DEFAULT 1204 .text._ZNK12opencv_caffe12NetParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   488: 0000000000000000     0 SECTION LOCAL  DEFAULT 1206 .text._ZN6google8protobuf2io17CodedOutputStream20WriteVarint64ToArrayEmPh
   489: 0000000000000000     0 SECTION LOCAL  DEFAULT 1207 .text._ZNK12opencv_caffe24DetectionOutputParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   490: 0000000000000000     0 SECTION LOCAL  DEFAULT 1209 .text._ZNK12opencv_caffe5Datum18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   491: 0000000000000000     0 SECTION LOCAL  DEFAULT 1211 .text._ZNK12opencv_caffe15FillerParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   492: 0000000000000000     0 SECTION LOCAL  DEFAULT 1213 .text._ZNK12opencv_caffe9ParamSpec18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   493: 0000000000000000     0 SECTION LOCAL  DEFAULT 1215 .text._ZNK12opencv_caffe13LossParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   494: 0000000000000000     0 SECTION LOCAL  DEFAULT 1217 .text._ZNK12opencv_caffe17AccuracyParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   495: 0000000000000000     0 SECTION LOCAL  DEFAULT 1219 .text._ZNK12opencv_caffe15ArgMaxParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   496: 0000000000000000     0 SECTION LOCAL  DEFAULT 1221 .text._ZNK12opencv_caffe15ConcatParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   497: 0000000000000000     0 SECTION LOCAL  DEFAULT 1223 .text._ZNK12opencv_caffe13BiasParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   498: 0000000000000000     0 SECTION LOCAL  DEFAULT 1225 .text._ZNK12opencv_caffe20ConvolutionParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   499: 0000000000000000     0 SECTION LOCAL  DEFAULT 1227 .text._ZNK12opencv_caffe13CropParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   500: 0000000000000000     0 SECTION LOCAL  DEFAULT 1229 .text._ZNK12opencv_caffe13DataParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   501: 0000000000000000     0 SECTION LOCAL  DEFAULT 1231 .text._ZNK12opencv_caffe30NonMaximumSuppressionParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   502: 0000000000000000     0 SECTION LOCAL  DEFAULT 1233 .text._ZNK12opencv_caffe16EltwiseParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   503: 0000000000000000     0 SECTION LOCAL  DEFAULT 1235 .text._ZNK12opencv_caffe16FlattenParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   504: 0000000000000000     0 SECTION LOCAL  DEFAULT 1237 .text._ZNK12opencv_caffe21InnerProductParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   505: 0000000000000000     0 SECTION LOCAL  DEFAULT 1239 .text._ZNK12opencv_caffe12LRNParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   506: 0000000000000000     0 SECTION LOCAL  DEFAULT 1241 .text._ZNK12opencv_caffe16PoolingParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   507: 0000000000000000     0 SECTION LOCAL  DEFAULT 1243 .text._ZNK12opencv_caffe18ReductionParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   508: 0000000000000000     0 SECTION LOCAL  DEFAULT 1245 .text._ZNK12opencv_caffe13ReLUParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   509: 0000000000000000     0 SECTION LOCAL  DEFAULT 1247 .text._ZNK12opencv_caffe16ReshapeParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   510: 0000000000000000     0 SECTION LOCAL  DEFAULT 1249 .text._ZNK12opencv_caffe14ScaleParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   511: 0000000000000000     0 SECTION LOCAL  DEFAULT 1251 .text._ZNK12opencv_caffe14SliceParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   512: 0000000000000000     0 SECTION LOCAL  DEFAULT 1253 .text._ZNK12opencv_caffe16SoftmaxParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   513: 0000000000000000     0 SECTION LOCAL  DEFAULT 1255 .text._ZNK12opencv_caffe13TileParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   514: 0000000000000000     0 SECTION LOCAL  DEFAULT 1257 .text._ZNK12opencv_caffe12SPPParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   515: 0000000000000000     0 SECTION LOCAL  DEFAULT 1259 .text._ZNK12opencv_caffe14NormalizedBBox18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   516: 0000000000000000     0 SECTION LOCAL  DEFAULT 1261 .text._ZNK12opencv_caffe21PSROIPoolingParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   517: 0000000000000000     0 SECTION LOCAL  DEFAULT 1263 .text._ZNK12opencv_caffe9BlobProto18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   518: 0000000000000000     0 SECTION LOCAL  DEFAULT 1265 .text._ZNK12opencv_caffe8NetState18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   519: 0000000000000000     0 SECTION LOCAL  DEFAULT 1267 .text._ZNK12opencv_caffe12NetStateRule18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   520: 0000000000000000     0 SECTION LOCAL  DEFAULT 1269 .text._ZNK12opencv_caffe11SolverState18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   521: 0000000000000000     0 SECTION LOCAL  DEFAULT 1271 .text._ZNK12opencv_caffe16V0LayerParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   522: 0000000000000000     0 SECTION LOCAL  DEFAULT 1273 .text._ZNK12opencv_caffe15SolverParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   523: 0000000000000000     0 SECTION LOCAL  DEFAULT 1275 .text._ZNK12opencv_caffe14LayerParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   524: 0000000000000000     0 SECTION LOCAL  DEFAULT 1277 .text._ZNK12opencv_caffe16V1LayerParameter18_InternalSerializeEPhPN6google8protobuf2io19EpsCopyOutputStreamE
   525: 0000000000000000     0 SECTION LOCAL  DEFAULT 1279 .text._ZN6google8protobuf8internal16InternalMetadataD2Ev
   526: 0000000000000000     0 SECTION LOCAL  DEFAULT 1281 .text._ZNK6google8protobuf11MessageLite21GetArenaForAllocationEv
   527: 0000000000000000     0 SECTION LOCAL  DEFAULT 1282 .text._ZN6google8protobuf8internal20RepeatedPtrFieldBase17MergeFromInternalERKS2_MS2_FvPPvS6_iiE
   528: 0000000000000000     0 SECTION LOCAL  DEFAULT 1284 .text._ZN12opencv_caffe9BlobShapeC2EN6google8protobuf8internal19ConstantInitializedE
   529: 0000000000000000     0 SECTION LOCAL  DEFAULT 1286 .text._ZN12opencv_caffe9BlobProtoC2EN6google8protobuf8internal19ConstantInitializedE
   530: 0000000000000000     0 SECTION LOCAL  DEFAULT 1288 .text._ZN12opencv_caffe15BlobProtoVectorC2EN6google8protobuf8internal19ConstantInitializedE
   531: 0000000000000000     0 SECTION LOCAL  DEFAULT 1290 .text._ZN12opencv_caffe16PermuteParameterC2EN6google8protobuf8internal19ConstantInitializedE
   532: 0000000000000000     0 SECTION LOCAL  DEFAULT 1292 .text._ZN12opencv_caffe22NormalizeBBoxParameterC2EN6google8protobuf8internal19ConstantInitializedE
   533: 0000000000000000     0 SECTION LOCAL  DEFAULT 1294 .text._ZN12opencv_caffe17PriorBoxParameterC2EN6google8protobuf8internal19ConstantInitializedE
   534: 0000000000000000     0 SECTION LOCAL  DEFAULT 1296 .text._ZN12opencv_caffe24DetectionOutputParameterC2EN6google8protobuf8internal19ConstantInitializedE
   535: 0000000000000000     0 SECTION LOCAL  DEFAULT 1298 .text._ZN12opencv_caffe5DatumC2EN6google8protobuf8internal19ConstantInitializedE
   536: 0000000000000000     0 SECTION LOCAL  DEFAULT 1300 .text._ZN12opencv_caffe15FillerParameterC2EN6google8protobuf8internal19ConstantInitializedE
   537: 0000000000000000     0 SECTION LOCAL  DEFAULT 1302 .text._ZN12opencv_caffe12NetParameterC2EN6google8protobuf8internal19ConstantInitializedE
   538: 0000000000000000     0 SECTION LOCAL  DEFAULT 1304 .text._ZN12opencv_caffe15SolverParameterC2EN6google8protobuf8internal19ConstantInitializedE
   539: 0000000000000000     0 SECTION LOCAL  DEFAULT 1306 .text._ZN12opencv_caffe11SolverStateC2EN6google8protobuf8internal19ConstantInitializedE
   540: 0000000000000000     0 SECTION LOCAL  DEFAULT 1308 .text._ZN12opencv_caffe8NetStateC2EN6google8protobuf8internal19ConstantInitializedE
   541: 0000000000000000     0 SECTION LOCAL  DEFAULT 1310 .text._ZN12opencv_caffe12NetStateRuleC2EN6google8protobuf8internal19ConstantInitializedE
   542: 0000000000000000     0 SECTION LOCAL  DEFAULT 1312 .text._ZN12opencv_caffe9ParamSpecC2EN6google8protobuf8internal19ConstantInitializedE
   543: 0000000000000000     0 SECTION LOCAL  DEFAULT 1314 .text._ZN12opencv_caffe14LayerParameterC2EN6google8protobuf8internal19ConstantInitializedE
   544: 0000000000000000     0 SECTION LOCAL  DEFAULT 1316 .text._ZN12opencv_caffe23TransformationParameterC2EN6google8protobuf8internal19ConstantInitializedE
   545: 0000000000000000     0 SECTION LOCAL  DEFAULT 1318 .text._ZN12opencv_caffe13LossParameterC2EN6google8protobuf8internal19ConstantInitializedE
   546: 0000000000000000     0 SECTION LOCAL  DEFAULT 1320 .text._ZN12opencv_caffe17AccuracyParameterC2EN6google8protobuf8internal19ConstantInitializedE
   547: 0000000000000000     0 SECTION LOCAL  DEFAULT 1322 .text._ZN12opencv_caffe15ArgMaxParameterC2EN6google8protobuf8internal19ConstantInitializedE
   548: 0000000000000000     0 SECTION LOCAL  DEFAULT 1324 .text._ZN12opencv_caffe15ConcatParameterC2EN6google8protobuf8internal19ConstantInitializedE
   549: 0000000000000000     0 SECTION LOCAL  DEFAULT 1326 .text._ZN12opencv_caffe18BatchNormParameterC2EN6google8protobuf8internal19ConstantInitializedE
   550: 0000000000000000     0 SECTION LOCAL  DEFAULT 1328 .text._ZN12opencv_caffe13BiasParameterC2EN6google8protobuf8internal19ConstantInitializedE
   551: 0000000000000000     0 SECTION LOCAL  DEFAULT 1330 .text._ZN12opencv_caffe24ContrastiveLossParameterC2EN6google8protobuf8internal19ConstantInitializedE
   552: 0000000000000000     0 SECTION LOCAL  DEFAULT 1332 .text._ZN12opencv_caffe20ConvolutionParameterC2EN6google8protobuf8internal19ConstantInitializedE
   553: 0000000000000000     0 SECTION LOCAL  DEFAULT 1334 .text._ZN12opencv_caffe13CropParameterC2EN6google8protobuf8internal19ConstantInitializedE
   554: 0000000000000000     0 SECTION LOCAL  DEFAULT 1336 .text._ZN12opencv_caffe13DataParameterC2EN6google8protobuf8internal19ConstantInitializedE
   555: 0000000000000000     0 SECTION LOCAL  DEFAULT 1338 .text._ZN12opencv_caffe30NonMaximumSuppressionParameterC2EN6google8protobuf8internal19ConstantInitializedE
   556: 0000000000000000     0 SECTION LOCAL  DEFAULT 1340 .text._ZN12opencv_caffe19SaveOutputParameterC2EN6google8protobuf8internal19ConstantInitializedE
   557: 0000000000000000     0 SECTION LOCAL  DEFAULT 1342 .text._ZN12opencv_caffe16DropoutParameterC2EN6google8protobuf8internal19ConstantInitializedE
   558: 0000000000000000     0 SECTION LOCAL  DEFAULT 1344 .text._ZN12opencv_caffe18DummyDataParameterC2EN6google8protobuf8internal19ConstantInitializedE
   559: 0000000000000000     0 SECTION LOCAL  DEFAULT 1346 .text._ZN12opencv_caffe16EltwiseParameterC2EN6google8protobuf8internal19ConstantInitializedE
   560: 0000000000000000     0 SECTION LOCAL  DEFAULT 1348 .text._ZN12opencv_caffe12ELUParameterC2EN6google8protobuf8internal19ConstantInitializedE
   561: 0000000000000000     0 SECTION LOCAL  DEFAULT 1350 .text._ZN12opencv_caffe14EmbedParameterC2EN6google8protobuf8internal19ConstantInitializedE
   562: 0000000000000000     0 SECTION LOCAL  DEFAULT 1352 .text._ZN12opencv_caffe12ExpParameterC2EN6google8protobuf8internal19ConstantInitializedE
   563: 0000000000000000     0 SECTION LOCAL  DEFAULT 1354 .text._ZN12opencv_caffe16FlattenParameterC2EN6google8protobuf8internal19ConstantInitializedE
   564: 0000000000000000     0 SECTION LOCAL  DEFAULT 1356 .text._ZN12opencv_caffe17HDF5DataParameterC2EN6google8protobuf8internal19ConstantInitializedE
   565: 0000000000000000     0 SECTION LOCAL  DEFAULT 1358 .text._ZN12opencv_caffe19HDF5OutputParameterC2EN6google8protobuf8internal19ConstantInitializedE
   566: 0000000000000000     0 SECTION LOCAL  DEFAULT 1360 .text._ZN12opencv_caffe18HingeLossParameterC2EN6google8protobuf8internal19ConstantInitializedE
   567: 0000000000000000     0 SECTION LOCAL  DEFAULT 1362 .text._ZN12opencv_caffe18ImageDataParameterC2EN6google8protobuf8internal19ConstantInitializedE
   568: 0000000000000000     0 SECTION LOCAL  DEFAULT 1364 .text._ZN12opencv_caffe21InfogainLossParameterC2EN6google8protobuf8internal19ConstantInitializedE
   569: 0000000000000000     0 SECTION LOCAL  DEFAULT 1366 .text._ZN12opencv_caffe21InnerProductParameterC2EN6google8protobuf8internal19ConstantInitializedE
   570: 0000000000000000     0 SECTION LOCAL  DEFAULT 1368 .text._ZN12opencv_caffe14InputParameterC2EN6google8protobuf8internal19ConstantInitializedE
   571: 0000000000000000     0 SECTION LOCAL  DEFAULT 1370 .text._ZN12opencv_caffe12LogParameterC2EN6google8protobuf8internal19ConstantInitializedE
   572: 0000000000000000     0 SECTION LOCAL  DEFAULT 1372 .text._ZN12opencv_caffe12LRNParameterC2EN6google8protobuf8internal19ConstantInitializedE
   573: 0000000000000000     0 SECTION LOCAL  DEFAULT 1374 .text._ZN12opencv_caffe19MemoryDataParameterC2EN6google8protobuf8internal19ConstantInitializedE
   574: 0000000000000000     0 SECTION LOCAL  DEFAULT 1376 .text._ZN12opencv_caffe12MVNParameterC2EN6google8protobuf8internal19ConstantInitializedE
   575: 0000000000000000     0 SECTION LOCAL  DEFAULT 1378 .text._ZN12opencv_caffe18ParameterParameterC2EN6google8protobuf8internal19ConstantInitializedE
   576: 0000000000000000     0 SECTION LOCAL  DEFAULT 1380 .text._ZN12opencv_caffe16PoolingParameterC2EN6google8protobuf8internal19ConstantInitializedE
   577: 0000000000000000     0 SECTION LOCAL  DEFAULT 1382 .text._ZN12opencv_caffe14PowerParameterC2EN6google8protobuf8internal19ConstantInitializedE
   578: 0000000000000000     0 SECTION LOCAL  DEFAULT 1384 .text._ZN12opencv_caffe15PythonParameterC2EN6google8protobuf8internal19ConstantInitializedE
   579: 0000000000000000     0 SECTION LOCAL  DEFAULT 1386 .text._ZN12opencv_caffe18RecurrentParameterC2EN6google8protobuf8internal19ConstantInitializedE
   580: 0000000000000000     0 SECTION LOCAL  DEFAULT 1388 .text._ZN12opencv_caffe18ReductionParameterC2EN6google8protobuf8internal19ConstantInitializedE
   581: 0000000000000000     0 SECTION LOCAL  DEFAULT 1390 .text._ZN12opencv_caffe13ReLUParameterC2EN6google8protobuf8internal19ConstantInitializedE
   582: 0000000000000000     0 SECTION LOCAL  DEFAULT 1392 .text._ZN12opencv_caffe16ReshapeParameterC2EN6google8protobuf8internal19ConstantInitializedE
   583: 0000000000000000     0 SECTION LOCAL  DEFAULT 1394 .text._ZN12opencv_caffe14ScaleParameterC2EN6google8protobuf8internal19ConstantInitializedE
   584: 0000000000000000     0 SECTION LOCAL  DEFAULT 1396 .text._ZN12opencv_caffe16SigmoidParameterC2EN6google8protobuf8internal19ConstantInitializedE
   585: 0000000000000000     0 SECTION LOCAL  DEFAULT 1398 .text._ZN12opencv_caffe14SliceParameterC2EN6google8protobuf8internal19ConstantInitializedE
   586: 0000000000000000     0 SECTION LOCAL  DEFAULT 1400 .text._ZN12opencv_caffe16SoftmaxParameterC2EN6google8protobuf8internal19ConstantInitializedE
   587: 0000000000000000     0 SECTION LOCAL  DEFAULT 1402 .text._ZN12opencv_caffe13TanHParameterC2EN6google8protobuf8internal19ConstantInitializedE
   588: 0000000000000000     0 SECTION LOCAL  DEFAULT 1404 .text._ZN12opencv_caffe13TileParameterC2EN6google8protobuf8internal19ConstantInitializedE
   589: 0000000000000000     0 SECTION LOCAL  DEFAULT 1406 .text._ZN12opencv_caffe18ThresholdParameterC2EN6google8protobuf8internal19ConstantInitializedE
   590: 0000000000000000     0 SECTION LOCAL  DEFAULT 1408 .text._ZN12opencv_caffe19WindowDataParameterC2EN6google8protobuf8internal19ConstantInitializedE
   591: 0000000000000000     0 SECTION LOCAL  DEFAULT 1410 .text._ZN12opencv_caffe12SPPParameterC2EN6google8protobuf8internal19ConstantInitializedE
   592: 0000000000000000     0 SECTION LOCAL  DEFAULT 1412 .text._ZN12opencv_caffe16V1LayerParameterC2EN6google8protobuf8internal19ConstantInitializedE
   593: 0000000000000000     0 SECTION LOCAL  DEFAULT 1414 .text._ZN12opencv_caffe16V0LayerParameterC2EN6google8protobuf8internal19ConstantInitializedE
   594: 0000000000000000     0 SECTION LOCAL  DEFAULT 1416 .text._ZN12opencv_caffe14PReLUParameterC2EN6google8protobuf8internal19ConstantInitializedE
   595: 0000000000000000     0 SECTION LOCAL  DEFAULT 1418 .text._ZN12opencv_caffe14NormalizedBBoxC2EN6google8protobuf8internal19ConstantInitializedE
   596: 0000000000000000     0 SECTION LOCAL  DEFAULT 1420 .text._ZN12opencv_caffe19ROIPoolingParameterC2EN6google8protobuf8internal19ConstantInitializedE
   597: 0000000000000000     0 SECTION LOCAL  DEFAULT 1422 .text._ZN12opencv_caffe17ProposalParameterC2EN6google8protobuf8internal19ConstantInitializedE
   598: 0000000000000000     0 SECTION LOCAL  DEFAULT 1424 .text._ZN12opencv_caffe21PSROIPoolingParameterC2EN6google8protobuf8internal19ConstantInitializedE
   653: 0000000000000000     0 SECTION LOCAL  DEFAULT 1506 .text._ZN12opencv_caffe9BlobShapeC2EPN6google8protobuf5ArenaEb
   656: 0000000000000000     0 SECTION LOCAL  DEFAULT 1510 .text._ZN12opencv_caffe9BlobProtoC2EPN6google8protobuf5ArenaEb
   658: 0000000000000000     0 SECTION LOCAL  DEFAULT 1513 .text._ZN12opencv_caffe15BlobProtoVectorC2EPN6google8protobuf5ArenaEb
   660: 0000000000000000     0 SECTION LOCAL  DEFAULT 1516 .text._ZN12opencv_caffe16PermuteParameterC2EPN6google8protobuf5ArenaEb
   663: 0000000000000000     0 SECTION LOCAL  DEFAULT 1520 .text._ZN12opencv_caffe22NormalizeBBoxParameterC2EPN6google8protobuf5ArenaEb
   665: 0000000000000000     0 SECTION LOCAL  DEFAULT 1523 .text._ZN12opencv_caffe17PriorBoxParameterC2EPN6google8protobuf5ArenaEb
   669: 0000000000000000     0 SECTION LOCAL  DEFAULT 1528 .text._ZN12opencv_caffe24DetectionOutputParameterC2EPN6google8protobuf5ArenaEb
   671: 0000000000000000     0 SECTION LOCAL  DEFAULT 1531 .text._ZN12opencv_caffe5DatumC2EPN6google8protobuf5ArenaEb
   673: 0000000000000000     0 SECTION LOCAL  DEFAULT 1534 .text._ZN12opencv_caffe15FillerParameterC2EPN6google8protobuf5ArenaEb
  ......
```

**Comment by Neumann-A** [comment]
a) this needs more information
b) I think you want a shared library instead of a static one?

**Comment by alexlee728** [comment]
I have developed a dynamic library A, which references the opencv static library. I want to hide the symbols of opencv and protobuf in the dynamic library (to avoid conflicts between programs using A and using protobuf). But I compiled the opencv static library and used "readelf - Ws libopencv_dnn4. a | grep protobuf" to see if it was hidden.

**Comment by alexlee728** [comment]
x64-linux

**Comment by cenit** [comment]
yes the problem came out while i was working on the 4.9 update. It will contain a fix for it #38928
